curbs/README.md

```diff
@@ -349,13 +349,15 @@ A rule defines who is allowed to do what, and for how long, on a curb, per the p
+| `max_stay_unit`  | Enum                                         | Optional          | The [Unit of Time](/general-information.md#unit-of-time-enum) associated with the `max_stay` value. Defaults to "minute".                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `no_return`      | Integer                                      | Optional          | The length of time (in units of `no_return_unit`) that a user must vacate a Curb Zone before being allowed to return for another stay. Defaults to 0. May not be applicable for all [activities](#activities).                                                                                                                                                                                                                                                                                                                                               |
+| `no_return_unit` | Enum                                         | Optional          | The [Unit of Time](/general-information.md#unit-of-time-enum) associated with the `no_return` value. Defaults to "minute".                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `user_classes`   | Array of [user class](#user-classes) Strings | Optional          | If specified, this regulation only applies to users matching the [user classes](#user-classes) contained within. If not specified, this regulation applies to everyone.                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Do you know why the descriptions of `user_classes` and `rate` have changed here? @zanebclark